src/papyrus-lang-vscode/src/features/commands/InstallDebugSupportCommand.ts

```diff
@@ -73,7 +73,7 @@ export class InstallDebugSupportCommand extends GameCommandBase {
             if (currentStatus === DebugSupportInstallState.installedAsMod) {
```

Comment:
If this is just treated as an override of the XSE path, we don't need a new install state. We can message what the full path happens to be regardless of the setting.

Reply:
Ok, I'll change that. I wasn't very familiar with the code when I originally added this.